app/models/customer.rb

```diff
@@ -36,6 +36,8 @@ class Customer < ApplicationRecord
   default_scope -> { kept }
   sequenced scope: ->(customer) { customer.organization.customers.with_discarded }
 
+  enum document_locale: I18n.available_locales
```

Comment:
I know it might have been covered in the dive-in, but what's the reason behind using an enum here, instead of a plain string type? 

My point is that if the order of `I18n.available_locales` we could end up in a really messed-up situation...
I would personally suggest to use the same approach as with countries and maybe with a custom validator. I think it will also be easier to read in the database.

Reply:
I see your point regarding I18n.available_locales reordering (and possible consequences if order changes with adding new locales) since we will have greater control by moving away from enums 👍  However, TBH not sure why not to use ISO language codes for `document_locale` field? If we use string words for language, we will also have to maintain mapping between language string and ISO language code... We can maybe switch from enum to string (with validator) but also use language iso code as value?

Reply:
Oh sure indeed, I was thinking at ISO code, like for countries or currencies :)

Reply:
updated